Packs/CrowdStrikeFalcon/Playbooks/playbook-CrowdStrike_Falcon_-_True_Positive_Incident_Handling.yml

```diff
@@ -2,10 +2,7 @@ id: CrowdStrike Falcon - True Positive Incident Handling
 version: -1
 fromversion: 6.5.0
 name: CrowdStrike Falcon - True Positive Incident Handling
-description:
-  This playbook is part of the 'Malware Investigation And Response' pack. For more information, refer to https://xsoar.pan.dev/docs/reference/packs/malware-investigation-and-response.
-  This playbook handles a CrowdStrike incident that was determined to be a true positive by the analyst.
-  Actions include isolating the host, blocking the indicator by the EDR, and tagging it.
+description: This playbook is part of the 'Malware Investigation And Response' pack. For more information, refer to https://xsoar.pan.dev/docs/reference/packs/malware-investigation-and-response. This playbook handles a CrowdStrike incident that was determined to be a true positive by the analyst. Actions include isolating the host, blocking the indicator by the EDR, and tagging it.
```

Comment:
General comment for the playbook - lets remove all those spaces between tasks)

Reply:
Done